### PR TITLE
Add events.win destination docs

### PR DESCRIPTION
**Pull request by megancooper**
### Proposed changes
* Adds documentation for the **events.win** destination

### Merge timing
ASAP once approved

### CODE REVIEW

File: src/connections/destinations/catalog/events.win/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: events.win Destination
+beta: true
+---
```
Comment (tcgilbert) [inline src/connections/destinations/catalog/events.win/index.md:4]:
```suggestion
---
title: events.win Destination
beta: true
id: 662d3328d029f89724a0c294
---
```

**Review by tcgilbert — CHANGES_REQUESTED** [review]
Just need to make the requested change, as well as change the folder name from `events.win` to `events-win`

**Comment by megancooper** [comment]
@tcgilbert Thank you! The folder name has been updated 🙏🏾

**Review by tcgilbert — APPROVED** [review]

File: src/connections/destinations/catalog/events-win/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+This destination is maintained by events.win. For any issues with the destination, [contact the events.win support team](mailto:hi@events.win).
+
+## Getting started
+
+1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="\_blank”} search for "events.win"
+2. Select events.win and click **Add Destination**
+3. Select an existing Source to connect to events.win (Actions).
+4. Go to the [events.win dashboard](https://app.events.win/developers){:target="\_blank"}, find and copy the **Developer key**.
+5. Enter the **Developer Key** in the events.win destination settings in Segment.
+6. That's it! events.win will start to receive data from Segment, there may be a delay before data is visible in the events.win dashboard.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/events-win/index.md:18]: hi @megancooper - do you have a feel for how long the maximum delay would be? 

```suggestion
1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="\_blank”} search for "events.win".
2. Select events.win and click **Add Destination**.
3. Select an existing Source to connect to events.win.
4. Go to the [events.win dashboard](https://app.events.win/developers){:target="\_blank"}, find and copy the **Developer key**.
5. Return to Segment and enter the **Developer Key** in the events.win destination settings.
6. events.win starts to receive data from Segment. There may be a delay before data is visible in the events.win dashboard.
```

File: src/connections/destinations/catalog/events-win/index.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Supported methods
+
+events.win supports the following methods, as specified in the [Segment Spec](/docs/connections/spec).
+
+### Track
+
+[Track](/docs/connections/spec/track) calls are consumed and validated against the tracking plan you've defined in events.win. We don't store the data, but we do provide a detailed report on how correct your data is.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/events-win/index.md:26]:
```suggestion
events.win consumes and validates [Track](/docs/connections/spec/track) calls against the tracking plan you've previously defined in events.win. events.win doesn't store the data, but instead provides a detailed report on how correct your data is.
```

File: src/connections/destinations/catalog/events-win/index.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="\_blank”} search for "events.win"
+2. Select events.win and click **Add Destination**
+3. Select an existing Source to connect to events.win (Actions).
+4. Go to the [events.win dashboard](https://app.events.win/developers){:target="\_blank"}, find and copy the **Developer key**.
+5. Enter the **Developer Key** in the events.win destination settings in Segment.
+6. That's it! events.win will start to receive data from Segment, there may be a delay before data is visible in the events.win dashboard.
+
+## Supported methods
+
+events.win supports the following methods, as specified in the [Segment Spec](/docs/connections/spec).
```
Comment (forstisabella) [inline src/connections/destinations/catalog/events-win/index.md:22]:
```suggestion
events.win supports Segment's [Track](/docs/connections/spec/track) method, as specified in the [Segment Spec](/docs/connections/spec).
```

File: src/connections/destinations/catalog/events-win/index.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+### Track
+
+[Track](/docs/connections/spec/track) calls are consumed and validated against the tracking plan you've defined in events.win. We don't store the data, but we do provide a detailed report on how correct your data is.
+
+You can use the [@events.win/cli](https://www.npmjs.com/package/@events.win/cli) to generate type-safe tracking code for your events.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/events-win/index.md:28]:
```suggestion
You can use the [@events.win/cli](https://www.npmjs.com/package/@events.win/cli){:target="\_blank”} to generate type-safe tracking code for your events.
```

**Review by forstisabella — APPROVED** [review]
a few small changes and we'll be ready to ship this 🚀

File: src/connections/destinations/catalog/events-win/index.md (hunk 2 above)
Comment (megancooper) [reply src/connections/destinations/catalog/events-win/index.md]: The delay at most would be a few minutes. I can add that detail in

**Pull request merged**